### PR TITLE
tiny page to explore public part of custom widget api

### PR DESCRIPTION
**Pull request by paulfitz**
A tiny helper custom widget to explore custom widget api calls. It'd be nice to grow something like this to look good and link with documentation.

### CODE REVIEW

File: inspect/api.html [hunk 1]
```diff
@@ -5,172 +5,328 @@
...
 </head>
 
 <body>
 
+  <div style="font-weight: 400;">
+    <br />
+    <i style="color:red">*Note*</i> Do not past code that you don't trust or is yours.
+  </div>   
+  <h4>
+    Feel free to explore the API and explore the API yourself. Each snippet is fully editable. <br />
```
Comment (paulfitz) [inline inspect/api.html:54]: explore the API and explore the API seems redundant

File: inspect/api.html [hunk 2]
```diff
@@ -5,172 +5,328 @@
...
   <h2>grist.ready()</h2>
-  <script>
-    async function gristReady(cmd) {
-      cmd.output("(already called, just needs to be called once)");
+  <div class="doc" contenteditable="true">
+    This is initial call a widget needs to make to let Grist know that
+    it is ready to process incoming events.
+  </div>
+  <form>
+    <script type="code">
```
Comment (paulfitz) [inline inspect/api.html:63]: Scrolling in this widget is awkward until you figure out the code boxes are capturing scroll events. Maybe they could capture scroll only if they have been clicked into?

File: inspect/api_deps.js [hunk 3]
```diff
@@ -0,0 +1,821 @@
+// Typescript definitions file bundled by dts-bundle manually.
+
+const definition = `
+// Generated by dts-bundle v0.7.3
```
Comment (paulfitz) [inline inspect/api_deps.js:4]: Could you write up the procedure for updating this?

**Review by berhalak — APPROVED** [review]
Thanks

**Pull request merged**